src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/UnprotectedAttributeTests.cs

```diff
@@ -268,6 +270,37 @@ private static void VerifyUnprotectedAttributes_Heterogenous3(byte[] encodedMess
         }
 
         [Fact]
+        [ActiveIssue(9252, PlatformID.Any)]
+        public static void TestUnprotectedAttributes1_EmptySet()
+        {
+            // This tests the behavior of unprotected attribute extraction when one of the attribute sequences declares an
+            // attribute type, but the contained SET OF AttributeValue is empty.
+            //Attribute ::= SEQUENCE {
+            //      attrType OBJECT IDENTIFIER,
+            //      attrValues SET OF AttributeValue }
+            // The encoded message was built in ASN.1 editor and tested in framework.It contains an enveloped message
+            // version 2 with a key transport recipient, the enveloped message contains data encrypted with 3DES and contains 2
+            // types of attribute. it contains two attributes of type document description and 0 of type document name. The important
```

Comment:
I'm not sure what you mean by "2 [of] document description and 0 [of] document name".  It might be more clear to say something like

```C#
// The attributes set is built as
// {
//     { attrType: document description, attrValues: { value1, value2 } },
//     { attrType: document name, attrValues: { } },
// }
```

And then make that be accurate.  And explain what Framework does vs what Core does.